news/2020-27-11-mappers-report-fall.md

```diff
@@ -0,0 +1,219 @@
+- **osu!taiko:** [Cynplytholowazy](https://osu.ppy.sh/users/3901754) & [Jerry](https://osu.ppy.sh/users/605973)
+- **osu!catch:** [Nelly](https://osu.ppy.sh/users/4741164), [Spectator](https://osu.ppy.sh/users/702598) & [Phob](https://osu.ppy.sh/users/6069462)
+- **osu!mania:** [Lirai](https://osu.ppy.sh/users/3734394), [Fresh Chicken](https://osu.ppy.sh/users/3984370) & [Unpredictable](https://osu.ppy.sh/users/7560872)
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open.
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com) and apply today!
+
+### BN Departures
+
+- **osu!:** [Venix](https://osu.ppy.sh/users/5999631), [Trynna](https://osu.ppy.sh/users/2652951), [[Karcher] 2](https://osu.ppy.sh/users/9892196), [NeKroMan4ik](https://osu.ppy.sh/users/11387664), [Ryuusei Aika](https://osu.ppy.sh/users/7777875), [Fall](https://osu.ppy.sh/users/4800816) & [Nikakis](https://osu.ppy.sh/users/4351739)
+- **osu!taiko:** [Roger](https://osu.ppy.sh/users/1538223),  [tasuke912](https://osu.ppy.sh/users/2774767), [Ulqui](https://osu.ppy.sh/users/1263669) & [incandescence](https://osu.ppy.sh/users/6256027)
+- **osu!catch:** [wonjae](https://osu.ppy.sh/users/5032045), [Sinnoh](https://osu.ppy.sh/users/4236057), [Bunnrei](https://osu.ppy.sh/users/829284) & [Baron](https://osu.ppy.sh/users/10286499)
+- **osu!mania:** [Keiga](https://osu.ppy.sh/users/6866022), [Garalulu](https://osu.ppy.sh/users/757783),  [Kamuy](https://osu.ppy.sh/users/7439226),  [Muse Dash](https://osu.ppy.sh/users/13695676) & [Kawawa](https://osu.ppy.sh/users/4647754)
+
+### NAT Additions
+
+A star shining brighter than the sun has ascended from the Beatmap Nominators and is shining now in a scary red color in the osu! sky. [hypercyte](https://osu.ppy.sh/users/9155377) is the newest addition to the Nomination Assessment Team! After two years of work in the Beatmap Nominators, he will now support the osu! NAT with the evaluation of applications and Beatmap Nominators to ensure a constant flow of new additions to the BNG.
+
+So once more: Welcome hypercyte!
+
+### NAT Retirement
+
+After the good news we sadly have a sad one for you. [Nepuri](https://osu.ppy.sh/users/6637817) has decided to take his leave. In over a year of work he helped the osu!taiko Beatmap Nominators to get back to their old strength. Now that he successfully accomplished this task, we hope he will find a fine and well deserved rest.
```

Comment:
"we sadly have a sad one" 🤔 

if that's not meant to sound silly choose different adjective